simtools/db_handler.py

```diff
@@ -568,7 +570,55 @@ def _getSiteParametersMongoDB(self, dbName, site, modelVersion, onlyApplicable=F
         )
 
         query = {
-            "Site": site,
+            "Site": _siteValidated,
+            "Version": _modelVersion,
+        }
+        if onlyApplicable:
+            query["Applicable"] = onlyApplicable
```

Comment:
Why don't you set here `quere["Applicable"] = True`, would be clearer? Or remove the `if onlyApplicable`?

Reply:
Changed to `query["Applicable"] = True`